release.yaml

```diff
@@ -1,5 +1,11 @@
 release:
   branch: develop
-  version: 7.6.0
-  nextVersion: 7.7.0-SNAPSHOT
+  version: 8.2.0
+  nextVersion: 8.3.0-SNAPSHOT
+  notesStartTag: 8.1.0
   mock: false
+  baseTag:
+    activiti: 8.2.0-alpha.25
+    activitiCloud: 8.2.0-alpha.35
+    commonChart: 8.2.0-alpha.4
+    fullChart: 8.2.0-alpha.44
```

Comment:
IMO, it's better to not include this latest part (`baseTag`) that's automatically generated when the rc version is created

Reply:
thanks, fixed